pkg/storage/accessibility_request.go

```diff
@@ -45,9 +44,7 @@ func (s *Store) CreateAccessibilityRequest(ctx context.Context, request *model.A
 		request,
 	)
 	if err != nil {
-		appcontext.ZLogger(ctx).Error(
-			fmt.Sprintf("Failed to create accessibility request with error %s", err),
-		)
+		appcontext.ZLogger(ctx).Error("Failed to create accessibility request with error %s", zap.Error(err))
```

Comment:
You don't need the `%s` anymore now that you're not in a `fmt.Sprintf`.

Reply:
Good catch! I will fix this in the next branch.